invenio_oaiserver/static/xsl/MARC21slimUtils.xsl

```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Given the comment on the `CC0`-licensing, I would add a comment header here at the beginning of the XML document with the licensing/copyright explanation and a link to the LoC website.